pipeline/src/default_priors.jl

```diff
@@ -0,0 +1,20 @@
+"""
+Constructs a dictionary of default prior distributions for the parameters used
+    in `EpiAware` models.
+
+# Returns
+- `Dict{String, Distribution}`: A dictionary containing the default prior
+    distributions.
+
+"""
+function default_priors()
```

Comment:
can we call this something a little clearer. its just shared latent model priors right?